src/librustdoc/html/static/js/source-script.js

```diff
@@ -48,6 +54,7 @@ function createDirEntry(elem, parent, fullPath, hasFoundFile) {
             const file = document.createElement("a");
             file.innerText = file_text;
             file.href = rootPath + "src/" + fullPath + file_text + ".html";
+            file.addEventListener("click", closeSidebarIfMobile);
```

Comment:
Instead of doing this, wouldn't it be simpler to not open the sidebar by default when we arrive on a new page with a "mobile width"?

Reply:
That makes sense indeed. Can you add a test where we check that a "resize" in-between won't re-open the sidebar as you mentioned?